buildenv

```diff
@@ -1,21 +1,24 @@
+LIBRDKAFKA_VERSION="2.2"
 export ZOPEN_GIT_URL="https://github.com/confluentinc/librdkafka.git"
 export ZOPEN_GIT_DEPS="git curl make pkgconfig openssl sed grep coreutils zoslib zlib zstd lz4"
 export ZOPEN_TYPE="GIT"
 export ZOPEN_RUNTIME_DEPS="bash"
 export ZOPEN_COMP="clang"
 
-export ZOPEN_EXTRA_CFLAGS="-fzos-le-char-mode=ascii -std=gnu11 -mnocsect -fno-short-enums -Wno-error -mzos-target=zosv2r4 -I../src"
+export ZOPEN_EXTRA_CFLAGS="-Wno-error -mzos-target=zosv2r4 -I../src"
 
-export ZOPEN_EXTRA_CPPFLAGS="-D_ZOS -D_OPEN_SYS_SOCK_IPV6 -D_UNIX03_SOURCE -D_UNIX03_THREADS -D_XOPEN_SOURCE_EXTENDED=1 -D_POSIX_C_SOURCE=200112L -D_POSIX_SOURCE -DPOSIX_SIGNALS -D_XOPEN_SOURCE=600 -mzos-target=zosv2r4 -I../src-cpp"
+export ZOPEN_EXTRA_CPPFLAGS="-D_OPEN_SYS_SOCK_IPV6 -D_UNIX03_SOURCE -D_UNIX03_THREADS -D_XOPEN_SOURCE_EXTENDED=1 -D_POSIX_C_SOURCE=200112L -D_POSIX_SOURCE -DPOSIX_SIGNALS -mzos-target=zosv2r4 -I../src-cpp"
 
-export LIBS="-L/u/ibmuser/zopen/prod/zoslib/lib -L/u/ibmuser/zopen/prod/openssl/lib -L/u/ibmuser/zopen/prod/zstd/lib -L/u/ibmuser/zopen/prod/lz4/lib -L../src -lzoslib -lrdkafka -llz4 -lzstd -lcrypto -lssl -lzoslib-supp"
+#export LIBS="-L/u/ibmuser/zopen/prod/zoslib/lib -L/u/ibmuser/zopen/prod/openssl/lib -L/u/ibmuser/zopen/prod/zstd/lib -L/u/ibmuser/zopen/prod/lz4/lib -L../src -lzoslib -lrdkafka -llz4 -lzstd -lcrypto -lssl -lzoslib-supp"
+export LIBS="-L../src"
```

Comment:
Does this override the LIBS that are implicitly added like zoslib?

Reply:
I verified it doesn't override implicitly added LIBS.